pkg/ccl/backupccl/restore_online.go

```diff
@@ -112,6 +125,29 @@ func sendAddRemoteSSTWorker(
 		var batchSize int64
 		const targetBatchSize = 440 << 20
 
+		// TODO(dt): handle tenants
+		prefixes := prefixRewriter{rewrites: make([]prefixRewrite, 0, len(tableRekeys))}
+		for _, i := range tableRekeys {
+
+			if i.OldID == 0 {
+				continue
+			}
+			var desc descpb.Descriptor
+			if err := protoutil.Unmarshal(i.NewDesc, &desc); err != nil {
+				return errors.Wrapf(err, "unmarshalling rekey descriptor for old table id %d", i.OldID)
+			}
+			table, _, _, _, _ := descpb.GetDescriptors(&desc)
+			if table == nil {
+				return errors.New("expected a table descriptor")
+			}
+
+			prefixes.rewrites = append(prefixes.rewrites, prefixRewrite{
+				OldPrefix: keys.SystemSQLCodec.TablePrefix(i.OldID),
+				NewPrefix: keys.SystemSQLCodec.TablePrefix(uint32(table.ID)),
```

Comment:
Just a note to self that we aren't handling tenants yet in this PR so this SystemSQLCodec is OK.

Reply:
Yeah, there's a `// TODO(dt): handle tenants` just above this block.